docs/preview/index.md

```diff
@@ -127,7 +137,7 @@ Visit https://localhost
 firefox https://localhost
 ```
 
-You now have a running instance of InvenioRDM at [https://localhost](https://localhost),
+You now have a running instance of InvenioRDM at [https://127.0.0.1](https://127.0.0.1),
```

Comment:
Just curious about the distinction here. Isn't localhost == 127.0.0.1 ?

Reply:
I am not sure of the policy, but `localhost` is forbidden in safari unless you clear cookies every single time (has to do with the self-signed certificate). However, it does not happen with 127.0.0.1

Reply:
Ok, we will see how people react and if we need to change the output produced by the command to say 127.0.0.1 as well.